components/bookingForm/bookingForm.tsx

```diff
@@ -123,7 +125,7 @@ export const BookingForm = ({ recaptchaKey }) => {
             {CONTACT_FORM_TITLE}
           </h2>
           {!!contactSuccess && (
-            <div className="alert alert-success mb-8" role="alert">
+            <div className={classNames(styles.alert, styles["alert-success"], "mb-8")} role="alert">
```

Comment:
given that this is the only place the classes are used - consider having the classes here instead of in the module.css